### ISSUE TITLE
카테고리별 페이지 구현방식 제안

### ISSUE THREAD

**Issue by hongsukchoi** [hyBlue/-13-SNUCSE-website-SWPP-#28]
카테고리 페이지 레이아웃 완성했습니다.
앞으로 소개, 구성원, 연구, 입학, 학사 부분을
사이드바만 조정해서 구현하면 될 것 같습니다.

구현방식은 구성원 카테고리를 예를 들면,
local:8080/members로 이동한다음, 사이드바나 top navbar의 세부카테고리(ex교수)를 눌렀을 때
기존처럼 /members/professors/ 식으로 새로운 페이지를 가져오는 게 아니라,
professor component를 members component로 로드하는 방식입니다.

다만 members/professors/ 라는 url이 더 이상 존재하지 않는 문제가 있는데,
antd 의 breadcumb 방식을 이용하면 될것같습니다.
https://ant.design/components/breadcrumb/
피드백 부탁드립니다.

**Comment by hyBlue** [comment]
네 이전보다 나은것 같습니다. 그리고 교수 페이지에서 디테일을 볼 때 사진을 좀 더 확대하거나 할 순 없나요? 사진이 너무 작은것 같습니다.
